src/sortStudents.ts

```diff
@@ -1,16 +1,76 @@
 
 export interface Student {
-  // describe Student interface
+  name: string,
+  surname: string,
+  age: number,
+  married: boolean,
+  grades: number[],
+
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name,
+  Surname,
+  Age,
+  Married,
+  AverageGrade,
+}
+
+function getAverageGrade(grades: number[]):number {
+  const average = grades.reduce((mark1, mark2) => mark1 + mark2);
+
+  return average / grades.length;
 }
```

Comment:
У тебя всё реализовано правильно, однако в методе редьюс рекомендуется добавлять начальное значение, в данном случае 0.

Также первый аргумент в коллбеке для редьюс не совсем оценка, это скорее накопитель, которому изначально присваивается начальное значение (в данном случае 0), и далее он накапливает в себе результаты итераций (здесь он накапливает оценки).

```suggestion
function getAverageGrade(grades: number[]):number {
  const average = grades.reduce((sum, mark) => sum + mark, 0);

  return average / grades.length;
}
```

Reply:
Хороший комент від Дениса, варто виправити